prow/apis/prowjobs/v1/types.go

```diff
@@ -607,6 +607,9 @@ func (g *GCSConfiguration) ApplyDefault(def *GCSConfiguration) *GCSConfiguration
 
 // Validate ensures all the values set in the GCSConfiguration are valid.
 func (g *GCSConfiguration) Validate() error {
+	if _, err := ExtractBucketName(g.Bucket); err != nil {
```

Comment:
`Checkconfig` needs to be extended to excercise this

Reply:
Done